python/paddle/v2/framework/tests/test_dynamic_recurrent_op.py

```diff
@@ -107,5 +111,58 @@ def test_forward(self):
         print 'pd_output', pd_output
 
 
+class RecurrentGradientOpTest(unittest.TestCase):
+    py = PyRNNStep()
+
+    def create_forward_op(self):
+        # create RNNOp
+        self.forward_op = DynamicRecurrentOp(
+            # inputs
```

Comment:
inlinks => inputs
boot_memories => initial_memory_value
step_net => step
outlinks => outputs
pre_memories => ex_memory

As you are making something innovative, I guess you don't want to copy †he terminology simply from Caffe2.

Reply:
done